Source/santad/SantadDeps.mm

```diff
@@ -109,21 +112,29 @@
     exit(EXIT_FAILURE);
   }
 
-  return std::make_unique<SantadDeps>([configurator metricExportInterval], esapi, std::move(logger),
-                                      control_connection, compiler_controller, notifier_queue,
-                                      syncd_queue, exec_controller, prefix_tree);
+  // TODO: `SNTMetricSet sharedInstance` should be injected
```

Comment:
This is done? Or am I missing something?

Reply:
Oops, old comment. Removed.